render_event_consts.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import "strings"
+
+func (renderer *Renderer) Render_event_consts() ([]byte, error) {
+	f := NewLineWriter()
+	f.WriteLine(`// GENERATED FILE: DO NOT EDIT!`)
+	f.WriteLine(``)
+	f.WriteLine(`package ` + renderer.Package)
+
+	for _, v := range renderer.Model.ComponentsSpecificationExtension {
+		if v.Value.Yaml != "" {
+
+			constants := strings.Split(v.Value.Yaml, "\n")
```

Comment:
Could you please make this logic (L12, L14, L16, L18) as a separate function and add there some unit tests for it.